code/backend/batch/BatchPushResults.py

```diff
@@ -27,21 +27,38 @@ def _get_file_name_from_message(msg: func.QueueMessage) -> str:
     arg_name="msg", queue_name="doc-processing", connection="AzureWebJobsStorage"
 )
 def batch_push_results(msg: func.QueueMessage) -> None:
-    do_batch_push_results(msg)
+    message_body = json.loads(msg.get_body().decode("utf-8"))
+    logger.info("Process Document Event queue function triggered: %s", message_body)
```

Comment:
for consistency can we please have f-strings in all places ? we have a scattered usage of regular strings ( like this line) and fstrings ( like line 45) ?

Reply:
I generally use `printf`-style %-formatting with loggers. Mainly for support with structured loggers like Sentry, but also for perf. For reference, nice explanation [here](https://blog.pilosus.org/posts/2020/01/24/python-f-strings-in-logging/).

However, I am happy to standardise either way. @ross-p-smith @adamdougal - is there a project preference?

Reply:
I don't believe there is a preference - I'm happy as long as we are consistent. When we decide, maybe a line in the CONTRIBUTING.md - we use " . . . "

Reply:
I'm happy either way. I'm not 100% familiar with the python logger, but loggers do sometimes do some nice formatting when using this approach